### ISSUE TITLE
VS code cannot connecting server on macOS

### ISSUE THREAD

**Issue by blinkbear** [microsoft/vscode-remote-release#7230]
- VSCode Version: 1.71.2 (Universal)
- Local OS Version:  macOS Monterey 12.5.1
- Remote OS Version: Ubuntu 20.04
- Remote Extension/Connection Type:  Remote Server
- Logs:  Just "connecting server ...."

Steps to Reproduce:

1.
2.

Does this issue occur when you try this locally?: Yes
Does this issue occur when you try this locally and all extensions are disabled?: No

**Comment by eleanorjboyd** [comment]
Hello! Could you supply a bit more information on your setup? Are you using ssh to connect? What extensions do you have enable? What are the full logs from `terminal` ->  `output` -> `remote-ssh`. Thanks!

**Comment by blinkbear** [comment]
Sorry for the delay in getting back to you. I used the remote tunnel to connect the  Visual Studio Coder Server. But there is NO remote-tunnel output from the terminal's output.  The vscode only shows the notification of 'connecting to <my-server>'. And finally, it failed with the output `Failed to connect to the remote extension host server (Error: WebSocket close with status code 1006)`. I have enabled 33 extensions in my workspace, including language support for C/C++, python, latex, and go; and remote ssh, Remote - SSH: Editing Configuration Files, Remote Development, Remote Explorer, and Remote Tunnels for connecting the remote server.
